src/config.py

```diff
@@ -120,7 +120,7 @@ def get_max_events_per_agent() -> int:
 
 # --- Core Settings ---
 DEBUG_MODE = get_env_var("DEBUG_MODE", required=False, default="false").lower() == "true"
-BASE_BRANCH = get_env_var("BASE_BRANCH", required=False, default="main")
+BASE_BRANCH = get_env_var("BASE_BRANCH", required=False, default=None)
```

Comment:
Should this still have a default? Or required=true?

Reply:
ya was debating this one and was hoping for some feedback. I think `required=true` would be better than default to `main`

Reply:
I set`required = True`